src/app/services/auth.service.ts

```diff
@@ -68,6 +71,9 @@ export class AuthService {
 
     logout() {
         if (this.isEnabled()) {
+            this.aerogear.apolloClient.clearStore();
```

Comment:
I think that logout can be called directly in auth view and then this circular dependency can be avoided. We can just have another method called from the screen called (wipe all data) and also add wipe all data button to settings as a bonus.

Reply:
I considered putting the logout steps in the actual Auth page instead of the auth service but I figured putting it in the page has more potential to cause bugs. What if we want to call `logout()` from somewhere else within the app for example? More than likely we would forget to call the appropriate thing. It also means that the auth page does not care about the implementation details of how logout works.